.circleci/config.yml

```diff
@@ -125,10 +124,7 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker:
-          reusable: true    # default - false
-          exclusive: true   # default - true
-          # Temporary workaround for Circle CI issue
-          # https://discuss.circleci.com/t/setup-remote-docker-connection-failures/26434
+          docker_layer_caching: false     # default - false
           version: 18.05.0-ce
```

Comment:
```suggestion
```
That comment relates to this line of code which I guess we can try to remove now.